system/autoware_diagnostic_graph_aggregator/doc/format/unit/diag.md

```diff
@@ -4,7 +4,9 @@ The `diag` object is a unit that refers to a specific status within the source d
 
 ## Format
 
-| Name   | Type     | Required | Description                            |
-| ------ | -------- | -------- | -------------------------------------- |
-| `type` | `string` | yes      | Specify `diag` when using this object. |
-| `diag` | `string` | yes      | The name of the diagnostic status.     |
+| Name      | Type     | Required | Description                            |
+| --------- | -------- | -------- | -------------------------------------- |
+| `type`    | `string` | yes      | Specify `diag` when using this object. |
+| `node`    | `string` | yes      | The name of the diagnostic status.     |
+| `name`    | `string` | yes      | The name of the diagnostic status.     |
+| `timeout` | `float`  | yes      | The name of the diagnostic status.     |
```

Comment:
(Same as my comment above)

Reply:
Same as https://github.com/autowarefoundation/autoware.universe/pull/10199#discussion_r1978592853, it is the arbitrary name part of the diagnostic status name.